### ISSUE TITLE
GitHub now has a built-in issue mover

### ISSUE THREAD

**Issue by octref** [google/github-issue-mover#147]
Just FYI: https://twitter.com/natfriedman/status/1057678371256119297

![image](https://user-images.githubusercontent.com/4033249/47806099-24aa3480-dcf6-11e8-94b1-49e3affc9590.png)

**Comment by AustinShalit** [comment]
There is still a place for this tool because GitHub's new feature does not allow you to transfer an issue outside of an organization.

**Comment by samtstern** [comment]
https://blog.github.com/changelog/2018-10-30-issue-transfer/

@nicolasgarnier

**Comment by DanielRosenwasser** [comment]
Yeah, pretty disappointed that this was deprecated because 2 years on I just hit the issue with cross-organizational issue transfer. I'll try to file an issue with GitHub.
